praw/models/modaction.py

```diff
@@ -1,6 +1,8 @@
 """Provide the ModAction class."""
+from six import string_types
```

Comment:
We only recently introduced websocket-client into PRAW. I understand six is well used, but I would prefer to not have explicit dependencies on it. I'm hoping at the end of the year to remove python2 support so I think it would be easier to do so by fixing a few if/else blocks in files, rather than removing six. So let's please not introduce uses of six in PRAW.

Reply:
Okay, I’ll remove uses of six, even thought I don’t really see good reasons not to…

> I think it would be easier to do so by fixing a few if/else blocks in files, rather than removing six.

… That doesn’t sound easier than renaming `string_types` to `str` (and then simply removing the six import)?